omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ProblemResource1_9.java

```diff
@@ -32,7 +34,8 @@
 	
 	@PropertySetter("problem")
 	public static void setProblem(Problem problem, Object value) {
-		problem.setProblem(new ConceptResource1_9().getByUniqueId((String) value));
+		String uuid = value instanceof Map ? ((Map) value).get("uuid").toString() : (String) value;
```

Comment:
Possibly you can add a utility function to do this since you are repeating it several times

Reply:
See my comment above, this needs to just go away